cla_backend/libs/eligibility_calculator/tests/cfe_civil/test_savings.py

```diff
@@ -21,21 +21,13 @@ def test_bank_balance_and_investments(self):
                     "subject_matter_of_dispute": False
                 },
             ],
-            "non_liquid_capital": [],
-        }}
-        self.assertEqual(expected, output)
-
-    def test_assets(self):
-        savings = Savings(bank_balance=0, investment_balance=0, asset_balance=80011)
-        output = translate_savings(savings)
-        expected = {"capitals": {
-            "bank_accounts": [],
             "non_liquid_capital": [
                 {
-                    "value": 800.11,
+                    "value": 2,
                     "description": "Asset",
                     "subject_matter_of_dispute": False
                 },
-            ]
+
+            ],
         }}
         self.assertEqual(expected, output)
```

Comment:
test_assets seemed useful, to show what happens when some inputs are zero, so not sure why this has been deleted?

Reply:
It was deleted due to the mis-understanding about assets_balance being liquid or otherwise, now restored